runtime/src/configs/mod.rs

```diff
@@ -0,0 +1,448 @@
+	pub const FileStorageEntryFee: Balance = 0;
+	pub const FileStorageFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(80),
+        community_treasury_percent: Percent::from_percent(20),
+        logion_treasury_percent: Percent::from_percent(0),
+        loc_owner_percent: Percent::from_percent(0),
+    };
+
+	pub const CertificateFee: Balance = 40 * MILLI_LGNT; // 0.04 LGNT per token
+    pub const CertificateFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(20),
+        community_treasury_percent: Percent::from_percent(80),
+        logion_treasury_percent: Percent::from_percent(0),
+        loc_owner_percent: Percent::from_percent(0),
+    };
+
+	pub const ValueFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(0),
+        community_treasury_percent: Percent::from_percent(0),
+        logion_treasury_percent: Percent::from_percent(100),
+        loc_owner_percent: Percent::from_percent(0),
+    };
+
+    pub const RecurentFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(0),
+        community_treasury_percent: Percent::from_percent(0),
+        logion_treasury_percent: Percent::from_percent(95),
+        loc_owner_percent: Percent::from_percent(5),
+    };
+
+    pub const IdentityLocLegalFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(0),
+        community_treasury_percent: Percent::from_percent(0),
+        logion_treasury_percent: Percent::from_percent(100),
+        loc_owner_percent: Percent::from_percent(0),
+    };
+
+    pub const OtherLocLegalFeeDistributionKey: DistributionKey = DistributionKey {
+        legal_officers_percent: Percent::from_percent(0),
+        community_treasury_percent: Percent::from_percent(0),
+        logion_treasury_percent: Percent::from_percent(0),
+        loc_owner_percent: Percent::from_percent(100),
+    };
+}
+pub type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
+
+pub struct RewardDistributor;
+impl logion_shared::RewardDistributor<NegativeImbalance, Balance, AccountId, RuntimeOrigin, LoAuthorityList>
+for RewardDistributor
+{
+	fn payout_community_treasury(reward: NegativeImbalance) {
+		if reward != NegativeImbalance::zero() {
+			Balances::resolve_creating(&CommunityTreasuryPalletId::get().into_account_truncating(), reward);
+		}
+	}
+
+	fn payout_logion_treasury(reward: NegativeImbalance) {
+		if reward != NegativeImbalance::zero() {
+			Balances::resolve_creating(&LogionTreasuryPalletId::get().into_account_truncating(), reward);
+		}
+	}
+
+	fn payout_to(reward: NegativeImbalance, account: &AccountId) {
+		if reward != NegativeImbalance::zero() {
+			Balances::resolve_creating(account, reward);
+		}
+	}
+}
```

Comment:
The `RewardDistributor` and distribution keys could be moved to their own module.